src/process.cpp

```diff
@@ -87,6 +93,21 @@ void process_init() {
     // unknownip = new Process (0, "", "unknown IP");
     // processes = new ProcList (unknownip, processes);
   }
+
+  if ( !(pidsToWatch.empty()) ) {
+    for (std::set<pid_t>::iterator it = pidsToWatch.begin(); it != pidsToWatch.end();) {
+      pid_t current_pid = *it;
+      if ( !(process_exists(current_pid))) {
```

Comment:
I believe that is interesting to check the process existence on **nethogs** start

Reply:
I'm not entirely convinced, but I'm not opposed either, so OK :)